src/js/index.js

```diff
@@ -64,4 +73,36 @@ profileButton.addEventListener('click', () => {
     profileButton.classList.add('active')
 
     header.textContent='Profile'
-})
\ No newline at end of file
+})
+
+textareaSections.forEach(textareaSection => {
+    setTextareaMaxLength(textareaSection)
+    addTextareaInputListener (textareaSection)
+})
+
+function setTextareaMaxLength (textareaSection) {
+    const textareaMaxlength = textareaSection.querySelector('[data-js="texarea-maxlength"]')
+    const textarea = textareaSection.querySelector('[data-js="createpage-textarea"]')
+    textareaMaxlength.innerHTML = textarea.maxLength
+    
+}
+
+function addTextareaInputListener (textareaSection) {
+    const textarea = textareaSection.querySelector('[data-js="createpage-textarea"]')
+    const textareaMaxlength = textareaSection.querySelector('[data-js="texarea-maxlength"]')
+    textarea.addEventListener('input', () => {
+        textareaMaxlength.innerHTML = textarea.maxLength - textarea.value.length+" / "+`${textarea.maxLength}`
```

Comment:
Please add spaces between math symbols for better readability (is Prettier not doing that?). Also: the last template string is not necessary. See:
```suggestion
        textareaMaxlength.innerHTML = textarea.maxLength - textarea.value.length + " / " + textarea.maxLength
```